### PR TITLE
doc: update ScyllaDB requirements - supported CPUs and AWS i4g instances

### PR DESCRIPTION
**Pull request by annastuchlik**
Fix https://github.com/scylladb/scylla-docs/issues/4144

### CODE REVIEW

File: docs/getting-started/system-requirements.rst [hunk 1]
```diff
@@ -216,6 +219,78 @@ See `ScyllaDB on the New AWS EC2 I4i Instances: Twice the Throughput & Lower Lat
 learn more about using ScyllaDB with i4i instances.
 
 
+i4g instances
```
Comment (tzach) [inline docs/getting-started/system-requirements.rst:222]: No need to list ARM base instances here
Im4gn and Is4gn are **supported**, but the recommended instances are only i3, i3en and i4i

Comment (avikivity) [reply docs/getting-started/system-requirements.rst:222]: Why shouldn't we list them?

They aren't named i4g though.

**Comment by tzach** [comment]
> Why shouldn't we list them?

Duplicating info from AWS, or anywhere else, is error prune and as a rule, should be avoided.
Better link to the relevant page.
An expectation is where the info is always needed, and you want to avoid the extra page jump.

**Review by tzach — APPROVED** [review]

**Comment by annastuchlik** [comment]
@avikivity I've made the updates. Could you re-review this PR?

**Comment by annastuchlik** [comment]
@avikivity Could you verify my changes so that this PR can be merged?

File: docs/getting-started/system-requirements.rst (hunk 1 above)
Comment (avikivity) [reply docs/getting-started/system-requirements.rst]: There are no i4g instances, there are im4gn and is4gen (IIRC, please check)

File: docs/getting-started/system-requirements.rst [hunk 2]
```diff
@@ -219,6 +219,78 @@ See `ScyllaDB on the New AWS EC2 I4i Instances: Twice the Throughput & Lower Lat
...
+     - 48
+     - 1 x 7500
+   * - Is4gen.4xlarge
+     - 16
+     - 96
+     - 2 x 7500
+   * - Is4gen.8xlarge
+     - 32
+     - 192
+     - 4 x 7500
```
Comment (avikivity) [inline docs/getting-started/system-requirements.rst]: Is there a reason to list all the instance types? This is duplicating AWS information. We should just link to that page.

Comment (annastuchlik) [reply docs/getting-started/system-requirements.rst]: I'm absolutely OK with just adding a link. @tzach Any objections? im4gn and is4gen are supported but nor recommended.

**Comment by annastuchlik** [comment]
@avikivity 
I've already removed the list of im4gn and is4gen instances, replacing them with the link to the AWS resource. Also, I've specified which instance types are recommended to improve clarity.
Is there anything else to be updated?

**Comment by annastuchlik** [comment]
@avikivity @tzach Do you think this PR can be merged? I've applied all your suggestions.

File: docs/getting-started/system-requirements.rst [hunk 3]
```diff
@@ -28,14 +28,17 @@ Scylla tries to maximize the resource usage of all system components. The shard-
 
 CPU
 ^^^
-Scylla requires modern Intel CPUs that support the SSE4.2 instruction set and will not boot without it.
+ScyllaDB requires modern Intel CPUs that support the SSE4.2 instruction set and will not boot without it.
```
Comment (mykaul) [inline docs/getting-started/system-requirements.rst:31]:
```suggestion
ScyllaDB requires modern Intel/AMD CPUs that support the SSE4.2 instruction set and will not boot without it.
```

**Comment by tzach** [comment]
LGTM

**Comment by annastuchlik** [comment]
Thanks. I think this PR can be merged, the content is all there. If any minor improvements are required, I can add them later.

**Comment by annastuchlik** [comment]
@scylladb/scylla-maint Please backport to 5.1

**Pull request merged**